docs/testplan/Chassis_GCU.md

```diff
@@ -0,0 +1,118 @@
+# Chassis/Multi-ASIC Generic Config Updater (GCU) Test Plan
+
+## Overview
+
+##  Generic Config Updater (GCU) Test Coverage for Multi-ASIC Support
+The existing community test coverage for the Generic Config Updater (GCU) has been enhanced to support running in multi-ASIC platforms and t2 topology.
+
+### Key Changes
+1. Multi-ASIC Adaptation:
+Test cases have been adapted to use a fixture that returns a random ASIC namespace for verification.
+
+2. Namespace Customization:
+For multi-ASIC DUT hosts, JSON patch paths are customized to include the namespace prefix. For changes in global paths, the /localhost prefix was added to the path.
+
+3. Single-ASIC Compatibility:
+For single-ASIC DUT hosts, no changes were made to the paths, ensuring compatibility with the existing test setup.
+
+### Test Case Updates
+
+- test_ipv6.py --> test_ip_bgp.py
+Previously, this test case verified paths only for IPv6 BGP neighbors. It has been expanded to run repeatedly for both IPv4 and IPv6 BGP neighbor types. As a result, the test suite file has been renamed to test_ip_bgp.py to reflect its broader scope.
+
+### Test PRs
+
+Above changes were added via Test PRs:
+- [Generalizing GCU test suite to verify both ip types (IPV4, IPV6) for BGP Neighbors](https://github.com/sonic-net/sonic-mgmt/pull/13650)
+- [GCU adding Multi-ASIC support in existing test code base](https://github.com/sonic-net/sonic-mgmt/pull/14070)
+
+## Testbed
+
+The test will run on T2 testbeds.
+
+## Setup Configuration
+
+TBD
+
+## Testing Plan
+
+### Test Case # 1 - Add Cluster With Basic QoS
+
+#### Test Objective
```

Comment:
Please add a case where we do this Add cluster interface config updates on an interface on Linecard1, and traffic is send from a different linecard ( say Linecard2) goes via this new interface. Basically inter linecard traffic is good and no drops